samples/test/placeholder_concat.py

```diff
@@ -17,21 +17,23 @@
 import kfp.v2.compiler as compiler
 
 component_op = components.load_component_from_text(
-    '''
+"""
 name: Component with concat placeholder
 inputs:
-- {name: input_prefix, type: String}
+- {name: input_one, type: String}
+- {name: input_two, type: String}
 implementation:
   container:
     image: gcr.io/google-containers/busybox
     command:
-    - echo
+    - sh
+    - -ec
     args:
-    - concat: [{inputValue: input_prefix}, 'some value']
-    '''
+    - concat: ['echo ', {inputValue: input_one}, '+', {inputValue: input_two}, '=three', ' > ', '/tmp/test ', '&&', " [[ $(cat /tmp/test) == 'one+two=three' ]]"]
```

Comment:
It might be a bit more readable if the script was separated from the concat args:
```
- sh
- ec
- echo "$0" > /tmp/test && [[ "$0" == 'one+two=three' ]]
- concat: [{inputValue: input_one}, '+', {inputValue: input_two}, '=three']
```

Reply:
this is a nice way to verify the sample

I'd personally prefer moving verification logic to the verify_func in _test.py, because I think verifying MLMD data has better coverage over the system. I'll build an example for it, this PR can keep going in this way.

Reply:
Done.